src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ConfigurationTests.cs

```diff
@@ -298,10 +356,12 @@ private string ConstructUserSettingsJson()
             return generatedUserSettings;
         }
 
-        private string ConstructExpectedOutput(bool redact)
+        private string ConstructExpectedOutput(bool redact, bool showSources = false)
```

Comment:
Maybe have a string parameter and make "Expected*" constants

Reply:
Just to clarify what you mean here - are you suggesting we pass in `directoryNameLocation` from the caller instead of determining it here using `showSources` (and that we save the two directory names as constants in `ConfigurationTests`)?

Reply:
I pushed an update that does what I think you're getting at - if you want it to be something different, just let me know before I check it in